CollAction/Frontend/app/donation/DebitDetailsBox.tsx

```diff
@@ -0,0 +1,128 @@
+﻿import * as React from "react";
+import { IdealBankElement, IbanElement } from 'react-stripe-elements';
```

Comment:
Single quote used here instead of double-quote. (same in DonationBox.tsx btw)

Reply:
Fixed